extensions/wikia/Helios/User.class.php

```diff
@@ -344,6 +344,16 @@ public static function onUserCheckPassword( $id, $username, $hash, $password, &$
 		return true;
 	}
 
+	/*
+	 * Invalidate all tokens for given user in helios service
+	 */
+	public static function onUserSetPassword( $userId ) {
+		$heliosClient = self::getHeliosClient();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)